package.json

```diff
@@ -31,14 +31,15 @@
     "rollup-plugin-node-resolve": "3.3.0",
     "rollup-plugin-progress": "^0.4.0",
     "rollup-plugin-screeps": "0.1.2",
-    "rollup-plugin-typescript2": "0.16.1",
+    "rollup-plugin-typescript2": "^0.21.1",
```

Comment:
I'm hesitant to make changes to dependencies, as it can break the checksum generation process. Is there a reason you are updating the rollup plugin?

Reply:
fixed, check latest PR